src/i_video.c

```diff
@@ -1635,6 +1636,8 @@ static void I_InitGraphicsMode(void)
 
     I_InitWindowIcon();
 
+    SDL_SetWindowResizable(screen, SDL_TRUE);
```

Comment:
What is the reasoning for this change?

Reply:
To cover this case:

> start the game in windowed mode then switch to borderless fullscreen

I'll shift the code around to make it clearer we're not setting the fullscreen window as resizable.